echo-notifications/src/main/groovy/com/netflix/spinnaker/echo/controller/NotificationController.groovy

```diff
@@ -53,7 +53,7 @@ class NotificationController {
   EchoResponse create(@RequestBody Notification notification) {
     notificationServices?.find {
       it.supportsType(notification.notificationType) &&
-        !notification.isInteractive() || it instanceof InteractiveNotificationService
+        (!notification.isInteractive() || it instanceof InteractiveNotificationService)
```

Comment:
This was it... 😝

Reply:
Thanks for fixing @luispollo but I got to say that I find this super hard to read and reason about. And if we fail to find a `notificationService` that satisfies the predicate, we just silently drop the notification? It seems like we should at the very least log an error message about it, maybe even emit a metric that we can get alerted about. What do you think?

Reply:
Hi @dreynaud, thanks for the feedback and sorry for the late reply. I'll add a logging statement there.

Reply:
P.S. In my defense, we were already dropping notifications that failed the previous check without logging... 😃 Not an excuse, though. I'll fix it.